assets/tests/components/map/controls/layersControl.test.tsx

```diff
@@ -14,7 +15,7 @@ const mapWrapper = ({ children }: { children: ReactNode }) => (
 
 describe("LayersControl", () => {
   test("when the control button is clicked, the list of layers is rendered with the selected layer marked as selected", async () => {
-    render(<LayersControl tileType="base" setTileType={jest.fn} />, {
+    render(<LayersControl tileType="base" setTileType={jest.fn()} />, {
```

Comment:
ooo thanks for fixing this! I'm curious, did the new types help you catch this at all?